core/include/traccc/utils/subspace.hpp

```diff
@@ -48,10 +48,6 @@ struct subspace {
         for (size_type i = 0u; i < kSize; ++i) {
             assert((indices[i] < kFullSize) and
                    "Axis indices must be within the full space");
-            if (0u < i) {
```

Comment:
Our measurement dimension is a run time variable so axis indices may not be unique for some cases. (e.g. `indices = {0,0}` for wire measurement)